packages/pros/src/feedforward.rs

```diff
@@ -0,0 +1,76 @@
+/// Feedforward controller for motor control.
+///
+/// This controller is used to apply feedforward control to achieve desired motor behavior
+/// based on velocity and acceleration.
+#[derive(Debug, Clone, Copy)]
```

Comment:
There might be issues with the feedforward being Copy. If it accidentally gets cloned, which could happen implicitly and without an error, then updating it in the future would cause the state to desynchronize between the copies.

```rs
let ff1 = FeedforwardController::new(...);
let ff2 = ff1; // might be an accident; user may have wanted to take by reference; updates from this point on won't match up

ff1.update(...);
```

I think it makes more sense to make the constants a separate struct that can be easily reused and cloned/copied (`FeedforwardConstants` or similar) while keeping the state hidden and removing the Copy implementation.

```rs
let ff1 = FeedforwardController::new(...);
let ff2 = FeedforwardController::from(ff1.constants()); // explicitly create new controller with different state

ff1.update(...);
```

I hope that makes sense :grin: